package.json

```diff
@@ -57,6 +57,7 @@
     "jest": "^26.6.1",
     "lerna-changelog": "^0.5.0",
     "lint-staged": "^9.2.0",
+    "lodash": "^4.17.20",
```

Comment:
Is this a leftover or?

Reply:
No, it's used in `Gulpfile.js` but since we didn't use PnP we didn't notice that it was missing :sweat_smile: